fw/http.c

```diff
@@ -5497,11 +5497,13 @@ tfw_http_req_cache_cb(TfwHttpMsg *msg)
 	goto conn_put;
 
 send_502:
-	tfw_http_send_err_resp(req, 502, "request dropped: processing error");
+	T_DBG("request dropped: processing error, status 502");
+	tfw_http_send_err_resp(req, 502, NULL);
 	TFW_INC_STAT_BH(clnt.msgs_otherr);
 	return;
 send_500:
-	tfw_http_send_err_resp(req, 500, "request dropped: processing error");
+	T_DBG("request dropped: processing error, status 500");
```

Comment:
I'm OK with this version, we already have a `tfw_http_send_err_resp()` with NULL `reason`. However, I'd prefer a way proposed by @const-t to introduce a function like `tfw_http_send_err_resp_nolog()`, which don't accept `reason` and which is called by `tfw_http_send_err_resp()` for the actual part of the sending

Reply:
Yes fixed